### PR TITLE
Stop using scorecard secret.

### PR DESCRIPTION
**Pull request by isarkis**
Scorecard analysis will still work without secret.  The only check that won't work is Branch Protection Rule check.

b/263488722

### CODE REVIEW

**Comment by andrewsavage1** [comment]
Could you explain why we want to get rid of this token?

**Comment by isarkis** [comment]
I don't want to create PAT under my account.  Can we create PAT for a bot account, something similar you've done with cherry picking mechanism?  In general, we are discouraged to use PATs.  The only benefit of using PAT in scorecard workflow is getting Branch Protection Mechanism check.

**Comment by andrewsavage1** [comment]
Yes, let's use a PAT from the existing bot account. Either we can reuse the existing token or create a new one.

File: .github/workflows/scorecards.yml [hunk 1]
```diff
@@ -41,7 +41,6 @@ jobs:
           results_file: results.sarif
           results_format: sarif
           # For more info check https://github.com/ossf/scorecard-action#authentication-with-pat.
```
[the remark this answers is not in the document]
Comment (isarkis) [reply .github/workflows/scorecards.yml]: done

**Pull request merged**